src/Microsoft.Extensions.Configuration.AzureAppConfiguration/Constants/LoggingConstants.cs

```diff
@@ -29,5 +29,8 @@ internal class LoggingConstants
 
         // Other
         public const string RefreshSkippedNoClientAvailable = "Refresh skipped because no endpoint is accessible.";
+        public const string RefreshFailedToGetSettingsFromEndpoint = "Failed to get configuration settings from endpoint";
+        public const string RefreshTryingToReadFromEndpoint = "Trying to read from endpoint";
+        public const string RefreshAllEndpointsFailed = "Failed to read from all endpoints.";
```

Comment:
This would be logged even if users set only one endpoint (no replicas). I think we can reuse RefreshFailedToGetSettingsFromEndpoint and log the previous endpoint that failed.

Reply:
Oh I see, I understood it as a message to show that all endpoints failed, but makes sense that we're missing a log for the last failed endpoint.